src/penn_chime/models.py

```diff
@@ -97,6 +106,27 @@ def __init__(self, p: Parameters) -> SimSirModel:
         self.daily_growth_t = daily_growth_helper(doubling_time_t)
 
 
+    def observed_predicted_loss(self, doubling_time: float, p: Parameters) -> float:
```

Comment:
The `doubling_time` parameter isn't actually used in this method

Reply:
Was the intention to set it in the mode, and run the simulation accordingly, and then look at `self.census_df`?

Reply:
thanks! these are details that will be radically different in the fix.

Reply:
Sorry, corrected typo mode->model

Reply:
out of date per new commits.